### DISCUSSION TITLE
Smoke detector not changing state during alarm

### DISCUSSION THREAD

**Discussion by jmerifjKriwe** [SukramJ/aiohomematic#1284] [General]
### I agree to the following

- [X] I have read the [documentation](https://github.com/danielperna84/custom_homematic#custom_homematic) right now
- [X] I have read the [FAQs](https://github.com/danielperna84/custom_homematic#frequently-asked-questions) right now
- [X] I am aware of the latest [release notes](https://github.com/danielperna84/custom_homematic/releases)
- [X] The Backend (CCU/Homegear/...) is working as expected and connected devices are controllable by its own UI.
- [X] I am running the latest version of the custom_component (and Home Assistant)

### The problem

Hello everyone,

I have an unpleasant issue with my smoke detectors. This morning, one of the smoke detectors triggered a false alarm. However, in HomeAssistant, the state (and the corresponding attribute) of the smoke detector did NOT change, but remained on "idle_off,". Additional the other smoke detectors that triggered the secondary alarm also remained in "idle_off" the entire time.

In RaspberryMatic, I can see that the triggering smoke detector's status changed at the corresponding time. So, it the alarm was received in RaspberryMatic as expected. But, as I mentioned, in HomeAssistant, nothing has been received. There is no corresponding entry in the logbook or history either.

My various other HmIP and (old) Hm-RF devices work without any problems, and changes in status, button presses, etc., are directly displayed in HomeAssistant.

Any idea what might be the reason for this? Ideally, in such a situation, I would like to execute various security actions (turn on lights, raise blinds, unlock doors, etc.). However, if the alarm is not received, the necessary trigger is missing.

Do I need to set up or consider anything specific for the HmIP-SWSD? Since the batteries are not the best, and my family is already annoyed due to the morning false alarm, I don't want to experiment too much with smoke tests.

Thank you for any ideas!

### What version of HomematicIP (local) has the issue?

1.48.0

### What type of installation are you running?

Home Assistant Container

### What type of installation are you running for your homematic backend?

RaspberryMatic Standalone

### Which version of your homematic backend are you running?

3.71.12.20231020

### Which config details do you use

- [X] Username and password
- [ ] TLS
- [ ] callback data ([see](https://github.com/danielperna84/custom_homematic#callback_host-and-callback_port))

### Which interfaces do you use?

- [X] Homematic IP
- [X] Homematic (Bidcos-RF)
- [ ] Groups (Heating-Group)
- [ ] BidCos-Wired (HM-Wired)

**Comment by SukramJ** [comment]
SMOKE_DETECTOR_ALARM_STATUS is a simple sensor, without any special logic behind it.
I can't tell you why it didn't work for you, and I'm not willing to test with my devices.
Unfortunately I can't find any errors on the integration side, so I'm moving the issue to the discussions. Maybe someone will give up their batteries.

**Comment by b0b1c** [comment]
hello, i have exactly the same issue. Did you solve it? Thanks
